src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -442,6 +444,18 @@ def generate_config(
         prometheus_user, prometheus_password = self.mgr._get_prometheus_credentials()
         FSID = self.mgr._cluster_fsid
 
+        clusters_credentials = {}
+        multi_cluster_config_raw = str(self.mgr.get_module_option_ex('dashboard', 'MULTICLUSTER_CONFIG'))
+        multi_cluster_config_str = multi_cluster_config_raw.replace("'", '"')
+        valid_multi_cluster_config_str = multi_cluster_config_str.replace('True', '"True"').replace('False', '"False"')
```

Comment:
is this implying the multi cluster config is not stored as a valid JSON object, but it's close enough you just modify it when you pull it out? Why is it not stored as something that can be json loaded immediately?

Reply:
multi-cluster config is stored as a JSON string with double quotes, since json.loads expects string with single quotes only, I have now used `ast.literal_eval` to load this is a json. Its there in the updated code